### PR TITLE
[WIP] Applied Visual Design Challenges

### PR DESCRIPTION
**Pull request by ghost**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/`.
- [ ] You have only one commit.
- [x] All new and existing tests pass the command `npm run test-challenges`.
#### Type of Change
- [x] New feature (non-breaking change which adds new functionality)
#### Checklist:
- [x] Tested changes locally.
- [x] Closes currently open issue: Closes FreeCodeCamp/CurriculumExpansion#7

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/applied-visual-design.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        "  <div class=\"caption\">",
+        "    <h3>Google</h3>",
+        "    <p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p>",
+        "    <span>Find out more <a href=\"https://en.wikipedia.org/wiki/Google\">here</a>.</span>",
+        "  </div>",
+        "",
+        "</div>"
+      ],
+      "tests": [
+        "assert($(\"h3\").css('text-align') == 'center', 'message: Center the <code>h3</code> tag.');",
```
Comment (atjonathan) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: Change these to:

> Your `h3` tag should be centered.

File: seed/challenges/01-responsive-web-design/applied-visual-design.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  "challenges": [
+    {
+      "id": "577af2a47f516b2611bb0674",
+      "title": "Create visual balance using the text-align property",
+      "description": [
+        "The <code>text-align</code> property in CSS is used to specify the alignment of text in an element. The most used values are <code>left</code>, <code>right</code>, <code>center</code> or <code>justify</code>.",
+        "Using <code>text-align</code>, align the following elements:",
+        "Center the <code>h3</code> tag.",
+        "Justify the <code>p</code> tag.",
+        "Align the <code>h5</code> element to the right."
```
Comment (Bouncey) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: There is no h5 in the seed

File: seed/challenges/01-responsive-web-design/applied-visual-design.json [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        "  <div class=\"caption\">",
+        "    <h3>Google</h3>",
+        "    <p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p>",
+        "    <span>Find out more <a href=\"https://en.wikipedia.org/wiki/Google\">here</a>.</span>",
+        "  </div>",
+        "",
+        "</div>"
+      ],
+      "tests": [
+        "assert($(\"h3\").height() == 40, 'message: Your <code>h3</code> should be <code>40px</code> long.');"
```
Comment (Bouncey) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: ~~long~~ heigh

File: seed/challenges/01-responsive-web-design/applied-visual-design.json (hunk 2 above)
Comment (atjonathan) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: Fixed

**Comment by atjonathan** [comment]
**Create a Visual Balance width and height property of an element (element initially exists out of balance among other elements).**

_**Description:**_
In the last challenges, you learned how to use the `height` and `width` properties in CSS.
Now, using those properties, give the `p` tag the same width and height as the image with the id of `avatar` so they are the same size.

_**Tests:**_
```js
assert($("img").height() == 486, 'message: Your <code>p</code> tag should have the height of <code>486px</code>.'),
assert($("img").width() == 320, 'message: Your <code>p</code> tag should have the height of <code>320px</code>.')
```

_**Initial Seed Code:**_
```html

<style>

  img {
    height: 486px;
    width: 320px;
  }

</style>

<div class="thumbnail">

  <img src="https://bit.ly/smallgooglelogo" alt="Google Logo">
  <div class="caption">
    <h3>Google</h3>
    <p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p>
    <img id="avatar" src="http://specials-images.forbesimg.com/imageserve/56ce2b24e4b062f6b59a7f7a/320x486.jpg?fit=scale&background=000000" />
  </div>

</div>
```

_**Solution:**_
```html

<style>

  img {
    height: 486px;
    width: 320px;
  }

  p {
    height: 486px;
    width: 320px;
  }

</style>

<div class="thumbnail">

  <img src="https://bit.ly/smallgooglelogo" alt="Google Logo">
  <div class="caption">
    <h3>Google</h3>
    <p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p>
    <img id="avatar" src="http://specials-images.forbesimg.com/imageserve/56ce2b24e4b062f6b59a7f7a/320x486.jpg?fit=scale&background=000000" />
  </div>

</div>
```

**Comment by atjonathan** [comment]
This PR will be reopened when we have the correct infrastructure in place :+1:

**Pull request closed without merging**